server/src/main/resources/evolutions/default/1.sql

```diff
@@ -28,3 +29,16 @@ CREATE TABLE user_logs (
 );
 
 CREATE INDEX user_logs_user_id_index ON user_logs USING BTREE (user_id);
+
+CREATE TABLE tokens (
```

Comment:
I'm yet to verify this.

Reply:
shouldn't we create a new evolution ?

Reply:
In this particular case, I don't mind, the template is still experimental and anyone who takes this template won't be able to just copy the evolution anyway (because there will be other conflicting evolutions)